### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by usmansbk**
# Project 2: Components.

In this pull request, I implemented the following

- Added a Calculator class component

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  {
+    text: '.',
+  },
+  {
+    text: '=',
+    className: 'bg-orange',
+  },
+];
+
+// eslint-disable-next-line react/prefer-stateless-function
```
Comment (arslanbisharat) [inline src/components/Calculator.js]: - [x] Please try to avoid unnecessary disabled eslint rules

File: src/App.js [hunk 2]
```diff
@@ -1,7 +0,0 @@
-const App = () => (
-  <div>
-    <h1>Welcome to Microverse</h1>
-  </div>
-);
-
-export default App;
```
Comment (arslanbisharat) [inline src/App.js]: - [x]  This file should be retrieved with a class-based component.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [x] There should be a file name `App.js` in the `src `directory of your project. That `App.js` file should have a **class-based** component as it is present in **components/calcuator.js**
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/Calculator.js (hunk 1 above)
Comment (usmansbk) [reply src/components/Calculator.js]: The linter will warn to convert to functional component without it.

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**